daml-script/test/daml/MultiTest.daml

```diff
@@ -39,3 +40,13 @@ partyIdHintTest = do
   alice <- allocatePartyWithHintOn "alice" (PartyIdHint "alice") (ParticipantName "one")
   bob <- allocatePartyWithHintOn "bob" (PartyIdHint "bob") (ParticipantName "two")
   pure (alice, bob)
+
+listKnownPartiesTest : Script ([PartyDetails], [PartyDetails])
+listKnownPartiesTest = do
+  parties1 <- listKnownPartiesOn (ParticipantName "one")
+  parties2 <- listKnownPartiesOn (ParticipantName "two")
+  p1 <- allocatePartyOn "p1" (ParticipantName "one")
+  p2 <- allocatePartyOn "p2" (ParticipantName "two")
+  parties1' <- listKnownPartiesOn (ParticipantName "one")
+  parties2' <- listKnownPartiesOn (ParticipantName "two")
```

Comment:
Shouldn't we have some assertions here as well? And do I get it right that we can't make assertions on the `isLocal` field here? I assume that doesn't stop us from asserting on the other two fields?

Reply:
We could but then we’re just verifying the same thing we already verify for the single participant tests which seems a bit pointless. `is_local` would be the interesting part to check that stuff is actually from different participants but given this issue there isn’t much that we can test herre.

Reply:
That makes sense. Thanks for the explanation.